CONTRIBUTING.md

```diff
@@ -33,15 +33,15 @@ Tecnócratas Digitales.
 
 * Elección de artículo.
     * Comenta de qué va a tratar.
-    * Asegurate que el artículo aporta valor.
+    * Asegúrate que el artículo aporta valor.
 * Redacción del artículo.
-    * Utiliza material propio.
-    * Si te basas en datos y artículos, añádelos en la sección de Bibliografía después del artículo.
+    * Utiliza material propio en la medida de lo posible.
+    * Si te basas en datos y artículos, añadelos en la sección de Bibliografía después del artículo.
     * Respeta las normas de linting.
 
 ### Guidelines de publicación
 
-* [ ] Sugiere el artículo al resto.
+* [ ] Sugiere el artículo en Tecnócratás Digitales.

```

Comment:
Por favor, revisad esta puntualización y marcad con un 👍 si estáis de acuerdo o con un 👎 si no lo estáis explicando las razones y/o proponiendo una sugerencia en una repuesta.